setup.py

```diff
@@ -70,7 +70,7 @@ def run_tests(self):
 ]
 
 DEV_EXTRAS = ['pyramid_debugtoolbar>=2.1', 'prospector[with_pyroma]', 'pep257',
-              'pyramid_multiauth', 'sphinxcontrib-httpdomain']
+              'pyramid_multiauth', 'sphinxcontrib-httpdomain', 'psycopg2']
```

Comment:
Given this is now in `DEV_EXTRAS` and in `requirements.txt`, I think we should probably move `psycopg2` and `psycogreen` both into setup.py as regular dependencies.